llvm/test/CodeGen/AMDGPU/mcexpr-knownbits-assign-crash.ll

```diff
@@ -0,0 +1,120 @@
+; REQUIRES: asserts
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx1030 -o - %s 2>&1 | FileCheck %s
+
+; CHECK-NOT: Assertion `BitWidth == RHS.BitWidth && "Bit widths must be same for comparison"' failed
```

Comment:
A check-not of an assert message is next to worthless, positively check the output